projects/demo/src/modules/app/pages.ts

```diff
@@ -360,6 +360,12 @@ export const pages: TuiDocPages = [
             },
         ],
     },
+    {
+        section: $localize`Components`,
+        title: 'InputFiles',
+        keywords: 'input-files, files, file, файлы',
+        route: '/components/files',
```

Comment:
maybe let's make it as a subpage of page `Inputs`

https://github.com/Tinkoff/taiga-ui/blob/6230118d701c43f15144a8d1356e4d502281891b/projects/demo/src/modules/app/pages.ts#L388-L390